extensions/s3/src/main/java/io/deephaven/extensions/s3/S3Instructions.java

```diff
@@ -32,14 +32,19 @@ public abstract class S3Instructions implements LogOutputAppendable {
     private final static Duration DEFAULT_CONNECTION_TIMEOUT = Duration.ofSeconds(2);
     private final static Duration DEFAULT_READ_TIMEOUT = Duration.ofSeconds(2);
 
+    static final S3Instructions DEFAULT_S3_INSTRUCTIONS = builder().build();
```

Comment:
I'm not sure how consistent we are in the code base, but I see DEFAULT used often.

```suggestion
    static final S3Instructions DEFAULT = builder().build();
```